### PR TITLE
Change adapter to static

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

## Proposed changes

- Change adapter to static to render all pages statically.
- Fix jampack
- Fix several css and html issues
- Improve video controls
- Improve a11y
- Improve webmanifest
- Serve fonts locally
- Serve css locally

## Types of changes

- [ ] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [x] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by maehr** [comment]
The build works, the settings changed due to the switch of the adapter: https://static-site.critique-digitale-ch.pages.dev/

**Comment by flicksolutions** [comment]
One general question:
Right now if you preview the build with `npm run build` you will not see the optimized version, because vite somehow builds from _.svelte-kit/output/_ I tried adding ` && jampack .\\.svelte-kit\\output` (windows-specific path just for tests) but then jampack doesn't find the images in the html (because they're in the client folder) and raises issues...
There are three options here:
1. I investigate further and see wether I can get vite to preview from the build folder directly (should work for static pages)
2. live with the errors that jampack generates until a _silent_ option is available
3. live with your preview not being optimized...

How should I proceed?

**Comment by maehr** [comment]
The optimization works just fine. You have to run `npm run postbuild` as well (which is done automatically by Cloudflare Pages). The adapter static outputs to the folder `build`. I adjusted `package.json` accordingly.

File: src/lib/components/Videos.svelte [hunk 1]
```diff
@@ -50,6 +50,10 @@
 
 		return `${minutes}:${seconds}`;
 	}
+
+	// Automatically hide controls after 2.5 seconds of inactivity
+	$: clearTimeout(showControlsTimeout);
```
Comment (flicksolutions) [inline src/lib/components/Videos.svelte:55]: this only works for me, if I delete this line... but without the line it works fine...

```suggestion
	
```

Comment (flicksolutions) [reply src/lib/components/Videos.svelte:55]: also: add `cursor: pointer;` on line 95.

**Review by flicksolutions — CHANGES_REQUESTED** [review]
The hiding of the controls doesn't work as proposed. With line 55 deleted, it works.
The proposed changes in package.json are only for DX.

**Comment by maehr** [comment]
@flicksolutions I introduced some bugs, please give me one more day before reviewing. Sorry

**Comment by maehr** [comment]
> @flicksolutions I introduced some bugs, please give me one more day before reviewing. Sorry

@flicksolutions done! I hope I fixed all requested issues.

**Pull request merged**